app_base.html

```diff
@@ -0,0 +1,47 @@
+<%inherit file = "/homepage/templates/app_base.htm" />
+<%! from catalog import models as cmod%>
+
+
+<%block name ="left_column">
+
+  <div id ='active_section'>
+      <ul id="category_list">
+
+      <li>
+          <a class = '${'activeone' if cat_name == 'All Products' else ''}' href="/catalog/index/0/">All Products</a>
+      </li>
+
+        % for categories in cmod.Category.objects.all():
+            <li>
+                <a class ='${ 'activeone' if cat_name == categories.name else ''}' href="/catalog/index/${ categories.id}/">${categories.name}</a>
+            </li>
+        % endfor
+      </ul>
+  </div>
+</%block>
+
+
+
+
+<%block name ="right_column">
+<h2 class="recent">Recently Viewed:</h2>
+<ol class ='pic_list'>
+  <%
+    if request.dmp.page == 'details':
```

Comment:
you may need to change this if we ever add more pages where the recently viewed list needs to show up

Reply:
Oh so it's not just in details, you mean?

Reply:
yes exactly. like if we add a secondary details page or something of the sort